### ISSUE TITLE
Learn Unity 5.x in parallel to C#, 

### ISSUE THREAD

**Issue by KatFishSnake** [HeartChess/Heartchess#2]
Spend like 2 to 3 evening learning Unity, 
- interface
- basic functions 
- 3D, 2D workflows
- Asset Generation
- Scripting
- simply saying anything you can find...

Thats  kind of essential part for us, but not a priority
